ios/DebugModule.swift

```diff
@@ -0,0 +1,33 @@
+import BitmovinPlayer
+
+@objc(DebugModule)
+public class DebugModule: NSObject, RCTBridgeModule {
+    // swiftlint:disable:next implicitly_unwrapped_optional
+    @objc public var bridge: RCTBridge!
+
+    // swiftlint:disable:next implicitly_unwrapped_optional
+    public static func moduleName() -> String! {
+        "DebugModule"
+    }
+
+    /// Module requires main thread initialization.
+    public static func requiresMainQueueSetup() -> Bool {
+        true
+    }
+
+    // swiftlint:disable:next implicitly_unwrapped_optional
+    public var methodQueue: DispatchQueue! {
+        bridge.uiManager.methodQueue
+    }
+}
+
+extension DebugModule {
+    /// Enable/disable verbose logging for the console logger.
+    /// - Parameter enabled: Whether to set verbose logging as enabled or disabled.
+    @objc(setLoggingEnabled:)
+    func setLoggingEnabled(enabled: Bool) {
```

Comment:
This should be `setDebugLoggingEnabled`, we only say logging

Reply:
Address in d1b0ed2232af581273c5e7a03380f27d484d4a5e , also for Android